--- src/app/practice/page.tsx
+++ src/app/practice/page.tsx
@@ -7,7 +7,7 @@
 import Pool3D from "@/components/Pool3D/Pool3D";
 import Settings, { NumberingDirection, SettingsValue, SimulationMode, StartingEnd } from "@/components/Settings/Settings";
 import { ph_event_swimulation } from "@/modules/phEvents";
-import { ISwimmer, SwimmerModel } from "@/modules/SwimmerModel";
+import { AVATARS, ISwimmer, SwimmerModel } from "@/modules/SwimmerModel";
 import { Button, ButtonGroup } from "@heroui/react";
 import { usePostHog } from "posthog-js/react";

@@ -44,7 +44,8 @@
     for (let i = 0; i < laps; i++) {
         lapTimes[i] += perLapDelta;
     }
-    return new SwimmerModel(lapTimes, Date.now());
+    const avatar = AVATARS[Math.floor(Math.random() * AVATARS.length)];
+    return new SwimmerModel(lapTimes, Date.now(), avatar);
 }

 export default function Page() {
