--- src/components/Swimmer/Swimmer.tsx
+++ src/components/Swimmer/Swimmer.tsx
@@ -4,15 +4,7 @@
 import { Container, Graphics, GraphicsContext, PointData, Sprite, Text, TextStyle } from "pixi.js";
-import { useRef, useState } from "react";
+import { useRef } from "react";
 import { getAnchor, getPosition } from "./swimmerUtils";

 extend({ Container, Graphics, GraphicsContext, Sprite, Text });
-const emojis = [
-    "🦈",  // MOR
-    "🩴",  // NCAC
-    "🏠",  // RSA
-    "🔱",  // TAC
-    "🌊",  // WAVE
-];

 export type SwimmerProps = {
@@ -29,8 +21,6 @@

 export default function Swimmer(props: SwimmerProps) {
-    // When first created, pick a random emoji from the list
-    const [useChar] = useState<string>(() => emojis[Math.floor(Math.random() * emojis.length)]);
     const textRef = useRef<Text>(null);

     useTick(() => {
@@ -46,7 +36,7 @@
     return (
         <pixiText
             ref={textRef}
-            text={useChar}
+            text={props.swimmer.avatar.emoji}
             style={new TextStyle({
                 fontSize: props.laneWidth * 0.5,
                 fill: "white", // Text color
